Author: Modestas Vainius <modax@debian.org>
Description: fix lookup of xscreensavers
 find_file(XSCREENSAVER_FILE_FOUND) sets CACHE value and does nothing if
 XSCREENSAVER_FILE_FOUND is set. Therefore, it must be unset for the next
 find_file() call to be effective after previous successful call.
Applied-Upstream: 4.6.1

--- a/kscreensaver/kxsconfig/CMakeLists.txt
+++ b/kscreensaver/kxsconfig/CMakeLists.txt
@@ -39,6 +39,7 @@ foreach(_currentdesktopfile ${_desktopfi
 	#MESSAGE(STATUS "newItem <${_newitem}>")
 	GET_FILENAME_COMPONENT(_screensaverName ${_newitem} NAME_WE)
 	#MESSAGE(STATUS "name without extension <${_screensaverName}>")
+   unset(XSCREENSAVER_FILE_FOUND CACHE)
 	find_file(XSCREENSAVER_FILE_FOUND ${_screensaverName}.xml PATHS ${XSCREENSAVER_CONFIG_DIR} )
 	if(XSCREENSAVER_FILE_FOUND)
 		MESSAGE(STATUS "xscreensaver name ${_screensaverName} found")
